PayloadPkg/Library/PayloadLib/PayloadLib.c

```diff
@@ -360,3 +360,62 @@ GetLoaderStage (
 {
   return LOADER_STAGE_PAYLOAD;
 }
+
+/**
+  Get device address
+
+  @param[in]  DeviceType         The device type, refer OS_BOOT_MEDIUM_TYPE.
+  @param[in]  DeviceInstance     The device instance number starting from 0.
+  @param[in]  PciMmioAddr        Only applicable to PCI device. TRUE will return
```

Comment:
Is PciMmioAddr useful ?   Why not always return the direct device address in the table.   If they need the PCIE MMIO address, they can always use macro to convert it.  I would recommend to make interface simpler.

Reply:
OK. will update it.